tasks/init-control-plane.yml

```diff
@@ -57,3 +57,13 @@
       --kubeconfig=/etc/kubernetes/admin.conf \
         > /tmp/flannel.log 2>&1
   changed_when: False
+
+- name: Check master node until ready
+  ansible.builtin.shell: |
+    kubectl get node {{ ansible_hostname }} \
```

Comment:
Change value according to this pr https://github.com/cloud-labs-infra/ansible-k8s-cluster/pull/53/files